src/ducks/userInfo/userInfoActions.jsx

```diff
@@ -27,6 +43,7 @@ export const fetchSignInData = (formData) => {
             })
             .catch(error => {
                 alert(error.response.data.message);
+                dispatch(errorCredentials(error.response));
```

Comment:
As I see, every error will be handled here. Why do you name it error**Credentials**?